Tests/ApiEndpoints/test_gordon360_pytest.py

```diff
@@ -321,136 +370,146 @@ def test_myschedule_put(self , session=None):
         self.url = hostURL + 'token'
-        self.token_payload = { 'username':username, 'password':password, 'grant_type':'password' }
+        self.token_payload = { 'username':username, 'password':password, \
+            'grant_type':'password' }
         response = api.post(self.session, self.url, self.token_payload)
 
         if not response.status_code == 200:
-            pytest.fail('Expected 200 OK, got {0}.'.format(response.status_code))
+            pytest.fail('Expected 200 OK, got {0}.'\
+                .format(response.status_code))
         try:
             response.json()
         except ValueError:
             pytest.fail('Expected Json, got {0}.'.format(response.text))
         if not 'access_token' in response.json():
-            pytest.fail('Expected access token in response, got {0}.'.format(response.json()))
+            pytest.fail('Expected access token in response, got {0}.'\
+                .format(response.json()))
+        assert response.json()["token_type"] == "bearer"
 
-#    Given valid credentials, verify that authentication is successful for a faculty/leader/god.
+#    Given valid credentials, verify that authentication is successful for a 
+#    faculty/leader/god.
 #    Endpoint --  token/
 #    Expected Status code -- 200 Ok
 #    Expected Content -- Json Object with access_token attribute.
     def test_authenticate_with_valid_credentials___activity_leader(self):
-        session = None
         self.session = requests.Session()
         self.url = hostURL + 'token'
-        self.token_payload = { 'username':leader_username, 'password':leader_password, 'grant_type':'password' }
+        self.token_payload = { 'username':leader_username, \
+            'password':leader_password, 'grant_type':'password' }
         response = api.post(self.session, self.url, self.token_payload)
         if not response.status_code == 200:
-            pytest.fail('Expected 200 OK, got {0}.'.format(response.status_code))
+            pytest.fail('Expected 200 OK, got {0}.'\
+                .format(response.status_code))
         try:
             response.json()
         except ValueError:
             pytest.fail('Expected Json, got {0}.'.format(response.text))
         if not 'access_token' in response.json():
-            pytest.fail('Expected access token in response, got {0}.'.format(response.json()))
+            pytest.fail('Expected access token in response, got {0}.'\
+                .format(response.json()))
+        assert response.json()["token_type"] == "bearer"
 
 class Test_allAccountTest(testCase):
-
-# # # # # # # # # # # #
-#    ACCOUNT TESTS    #
-# # # # # # # # # # # #
+# # # # # # # # #
+# ACCOUNT TESTS #
+# # # # # # # # #
 
 #    Verify that a user can get account by email
 #    Endpoint -- api/accounts/email/{email}
 #    Expected Status Code -- 200 OK
-#    Expected Respones Body -- profile of the email person 
+#    Expected Response Body -- profile of the email person
+#    Security risk 
```

Comment:
What does "Security risk" mean?  (I see it again below...)

Reply:
I wrote more to the comments. They are about accessing Gordon ID and barcodes from just a user's Gordon email.